packages/nimble-blazor/NimbleBlazor/Components/NimbleDialog.razor.cs

```diff
@@ -55,7 +55,7 @@ public partial class NimbleDialog<TCloseReason> : ComponentBase
     public async ValueTask<DialogResponse<TCloseReason>> ShowAsync()
     {
         // Pass cancellation token to disable default async timeout
-        CancellationTokenSource source = new CancellationTokenSource();
+        CancellationTokenSource source = new();
```

Comment:
Is this preferred over: 
```
var source = new CancellationTokenSource();
```

Reply:
[Simplify new expression (IDE0090)](https://learn.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0090)

This rule defaults to "Prefer target-typed new expressions when created type is apparent", but it's configurable. I didn't feel strongly on this one either way.

Reply:
Eh, I'm indifferent myself.